package.json

```diff
@@ -6,19 +6,21 @@
     "dev": "sapper dev",
     "build": "sapper build",
     "export": "sapper export",
-    "start": "sapper start",
+    "start": "node build",
```

Comment:
Is this correct? shouldn't this be `npm build`?

Reply:
it's correct, yeah — as of 0.11, Sapper creates a launcher file which defaults to `build/index.js`. So to run the app you just to `node build`

Reply:
This probably explains my problems w/ trying to build when upgrading Sapper. ;) Need to upgrade `sapper-template` too...

Reply:
I got it all working.